### DISCUSSION TITLE
Distorted PDF to PPT conversion

### DISCUSSION THREAD

**Discussion by AkshatSparrow** [Stirling-Tools/Stirling-PDF#1267] [Q&A]
I am using QuestPDF to generate a PDF.
When the PDF is generated on windows(localhost), StirlingPDF is able to convert this PDF into PPT properly,
but when the PDF is generated on linux (server),  the PDF-to-PPT conversion is improper,
the text throughout the PPT shifts to left side.
I am not sure but this might have something to do with the font. I have tried with multiple fonts till now , but all in vain.
Can anyone explain what is happening?
Have attached the screenshots for reference.

PDF
![Screenshot 2024-05-22 095025](https://github.com/Stirling-Tools/Stirling-PDF/assets/133651362/3360445b-58be-47e5-89fd-ab8d01aea16c)

PPT
![Screenshot 2024-05-22 095503](https://github.com/Stirling-Tools/Stirling-PDF/assets/133651362/845ccf7f-a15b-4d88-aea4-2e3fe4149293)

**Comment by Frooodle** [comment]
Are both docker instances or one isn't or?

**Reply by AkshatSparrow** [reply]
I am using the same docker instance in both the cases.

**Reply by AkshatSparrow** [reply]
@Frooodle Did you got anything on this?

**Reply by Frooodle** [reply]
Can you try 0.25.6, this uses new docker image which might have new results

**Reply by Frooodle** [reply]
If that doesnt work nothing we can do sorry, its a issue based on libreoffice, 3rd party and nothing we can do
